deepinv/physics/tomography.py

```diff
@@ -32,6 +33,11 @@ class Tomography(LinearPhysics):
     :param bool circle: If ``True`` both forward and backward projection will be restricted to pixels inside a circle
         inscribed in the square image.
     :param bool parallel_computation: if True, all projections are performed in parallel. Requires more memory but is faster on GPUs.
+    :param bool adjoint_via_backprop: if True, the adjoint will be computed via ``deepinv.physics.adjoint_function``. Otherwise the inverse Radon transform is used.
```

Comment:
```suggestion
    :param bool adjoint_via_backprop: if `True`, the adjoint will be computed via :func:`deepinv.physics.adjoint_function`. Otherwise the inverse Radon transform is used.
```